### ISSUE TITLE
Sidenav collapse menu item cannot be focused

### ISSUE THREAD

**Issue by gnomeontherun** [clr-team/planning#63]
## Description of work

The sidenav collapsable nav menu titles are not focusable with keyboard. They should be focusable so they can be interacted with keyboard only.

## Issue(s)

* https://github.com/vmware/clarity/issues/3526

## Reviewers

* Bozhidar
* Colin
* Cory
* Ivan
* Jeeyun
* Jeremy
* Matt
* Scott
* Shijir

## Definition of done

* [ ] Tests completed
* [ ] Reviewed by peers
* [ ] Design sign off (if necessary)
* [ ] Merged into master/topic branch
* [ ] Backport (if necessary)

**Comment by gnomeontherun** [comment]
Closing in favor of #93
